commit e1be22123ec086d42cf5f3405e4eb832200a6f69 (refs/changes/87/887/3)
Author: Alex Wilson <alex.wilson@joyent.com>
Date:   2016-11-22T00:19:25+00:00 (2 years, 11 months ago)
    
    joyent/node-cueball#48 use higher DNS concurrency on bootstrap
    Reviewed by: Cody Peter Mello <cody.mello@joyent.com>

diff --git a/lib/resolver.js b/lib/resolver.js
index a5e6562..aa0474f 100644
--- a/lib/resolver.js
+++ b/lib/resolver.js
@@ -309,7 +309,8 @@ CueBallDNSResolver.prototype.state_check_ns = function (S) {
 				service: '_dns._udp',
 				defaultPort: 53,
 				log: this.r_log,
-				recovery: this.r_recovery
+				recovery: this.r_recovery,
+				maxDNSConcurrency: 10
 			});
 			this.r_bootstrap = res.r_fsm;
 			CueBallDNSResolver.bootstrapResolvers[notIp[0]] =
diff --git a/package.json b/package.json
index 5430225..ddd4283 100644
--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "cueball",
-  "version": "1.1.4",
+  "version": "1.1.5",
   "description": "",
   "main": "lib/index.js",
   "dependencies": {
